lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -0,0 +1,653 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Loic Blot (@nerzhul) <loic.blot@unix-experience.fr>
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: postgresql_publication
+short_description: Add, update, or remove PostgreSQL publication
+description:
+- Add, update, or remove PostgreSQL publication.
+version_added: "2.9"
+options:
+  name:
+    description:
+    - Name of the publication to add, update, or remove.
+    required: true
+    type: str
+  db:
+    description:
+    - Name of the database to connect to and where
+      the publication state will be changed.
+    aliases: [ login_db ]
+    type: str
+  tables:
+    description:
+    - List of tables to add to the publication.
+    - If no value is set all tables are targeted.
+    - If the publication already exists for specific tables and I(tables) is not passed,
+      nothing will be changed. If you need to add all tables to the publication with the same name,
+      drop existent and create new without passing I(tables).
+    type: list
+  state:
+    description:
+    - The publication state.
+    default: present
+    choices: [ absent, present ]
+    type: str
+  parameters:
+    description:
+    - Dictionary with optional publication parameters.
+    - Available parameters depend on PostgreSQL version.
+    type: dict
+  owner:
+    description:
+    - Publication owner.
+    - If I(owner) is not defined, the owner will be set as I(login_user) or I(session_role).
+    type: str
+  cascade:
+    description:
+    - Drop publication dependencies. Has effect with I(state=absent) only.
+    type: bool
```

Comment:
Should this have `default: false`?

Reply:
exactly, ```false``` is the default behaviour, isn't it? 
(if we're talking about logic, yes, it should be false because of ```cascade``` option is a potentially dangerous operation in each command where it is)

Reply:
The default is `None`, and in the docs no default will be shown. Especially since it is dangerous, I would show to the user explicitly that the default is `false`.

Reply:
clear! thank you for explaining! )

Reply:
@felixfontein the guys suggested removing all ```default: false``` lines from my first modules, so I decided that it is one of the best practicies to do things in this way.
Now i see. i'll fix it all of the modules where necessary.

Reply:
I don't see why `default: false` should not be there. At least here :) Depends a bit on what kind of option it is...

What definitely shouldn't be there is `required: false`, but that's another option...

Reply:
yep, agree. In this case clear is much better than unclear